CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.0.0
```

Comment:
Is this a breaking change?

Reply:
It's dependent on #1746 to get rid of the gulp issue identified by `npm audit`, hence why I'm targeting `v4.0.0`.

Reply:
Ah ok - FYI we will not lose gulp immediately as we still need it for legacy lint tasks :( however I don't think this poses any risk

Reply:
4.1.0?